test/nodes/local-simulator_spec.js

```diff
@@ -9,11 +14,38 @@ describe('LocalSimulatorNode', function() {
   });
 
   afterEach(function(done) {
+    shell.stop();
     nodeTestHelper.unload();
     nodeTestHelper.stopServer(done);
   });
 
   it('load node', function(done) {
     testUtil.isLoaded(localSimulatorNode, 'local-simulator', done);
   });
+
+  it('execute script', function(done) {
+    const flow = new FlowBuilder();
+    flow.add('quantum-circuit', 'n0', ['n1'], {structure: 'qubits', outputs: '1', qbitsreg: '1', cbitsreg: '1'});
+    flow.add('hadamard-gate', 'n1', ['n2']);
+    flow.add('measure', 'n2', ['n3'], {selectedBit: '0'});
+    flow.add('local-simulator', 'n3', ['n4'], {shots: '1'});
+    flow.addOutput('n4');
+
+    nodeTestHelper.load(flow.nodes, flow.flow, function() {
+      let n0 = nodeTestHelper.getNode('n0');
+      let n4 = nodeTestHelper.getNode('n4');
+
+      n4.on('input', function(msg) {
+        try {
+          let command = util.format(snippets.LOCAL_SIMULATOR, '1');
+          assert.strictEqual(shell.lastCommand, command);
+          done();
+        } catch (err) {
+          done(err);
+        }
+      });
+
+      n0.receive({payload: ''});
+    });
+  });
```

Comment:
Here is an example of how `FlowBuilder` can be used to create a flow:
- Initialise a new flow with `flow = new FlowBuilder()`.
- Add nodes to the flow with `flow.add(...)`. The first parameter of this method is the name of the node, the second parameter is the ID of the node, the third parameter is an array of strings which contain the IDs of the connected output nodes, and the fourth (optional) parameter is an object containing any additional parameters you want to pass to the node (for example, configuration settings).
- To get the final output of the flow, use the `flow.addOutput` method. This uses a special `helper` node provided by the Node-RED Test Helper. Usually this node will be placed at the end of the flow.
- The line `nodeTestHelper.load(flow.nodes, flow.flow, function() { ...` will load the flow. `flow.nodes` contains all of the nodes to be loaded at runtime, whilst `flow.flow` is the JSON representation of the flow to be loaded.
- For nodes you're interested in doing something with, retrieve the node with the `nodeTestHelper.getNode(...)` method.
- You can then add an input listener to the node or send data to it as a payload.

For clarification, the following image is the visual representation of the same flow being created using the `FlowBuilder`. Note that we don't need to add the inject node to the `FlowBuilder` flow, since the line `n0.receive({payload: ''})` is what we use to send input instead.
![simple-circuit](https://user-images.githubusercontent.com/48141481/129645982-6af8245b-ae08-499f-b830-64230b252bd4.png)